src/PrestaShopBundle/Cache/LocalizationWarmer.php

```diff
@@ -54,20 +54,28 @@ public function warmUp($cacheDir)
             }
         }
 
-        $path_cache_file = $cacheDir.$this->version.$this->country.'.xml';
+        $path_cache_file = $cacheDir . $this->version . $this->country . '.xml';
 
         if (is_file($path_cache_file)) {
             $localization_file_content = file_get_contents($path_cache_file);
         } else {
-            $localization_file_content = @Tools::file_get_contents('http://api.prestashop.com/localization/'.$this->version.'/'.$this->country.'.xml');
-            if (!@simplexml_load_string($localization_file_content)) {
+            if (!defined('_PS_IN_TEST_')) {
+                $localization_file_content = @Tools::file_get_contents(
+                    'http://api.prestashop.com/localization/' . $this->version . '/' . $this->country . '.xml'
```

Comment:
if api go to https, this call will fail: can you make it configurable? Maybe using a container parameter?

Reply:
Downloading this file is no more wanted anyway. I'll just delete this part.